app/scripts/templates/complete_account_unlock.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content">
+  {{#isLinkExpired}}
+  <header>
+    <h1 id="fxa-account-unlock-link-expired-header">{{#t}}Unlock link expired{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+
+    <p>
+      {{#t}}The link you clicked to verify your email is expired.{{/t}}
```

Comment:
This is the same wording used in [complete_sign_up](https://github.com/mozilla/fxa-content-server/blob/bce776bbc2a1d6852e9f7d991d770a8ffb40a045/app/scripts/templates/complete_sign_up.mustache#L12), though "has" does sound nicer when I say it in my head.